.dagger/test.go

```diff
@@ -41,6 +41,99 @@ func (t *Test) All(
 	return t.test(ctx, "", "", "./...", failfast, parallel, timeout, race, 1, verbose)
 }
 
+// Run telemetry tests
+func (t *Test) Telemetry(
+	ctx context.Context,
+	// Only run these tests
+	// +optional
+	run string,
+	// Skip these tests
+	// +optional
+	skip string,
+	// +optional
+	update bool,
+	// +optional
+	failfast bool,
+	// +optional
+	parallel int,
+	// +optional
+	timeout string,
+	// +optional
+	race bool,
+	// +default=1
+	count int,
+	// +optional
+	verbose bool,
+) (*dagger.Directory, error) {
+	engine := t.Dagger.Engine().
```

Comment:
nit: a lot of this logic seems shared with `testCmd`?

Reply:
Yeah could probably be refactored or done differently - I didn't want _all_ of `testCmd` because I wanted a faster feedback loop and that does some other stuff I don't need, like building an engine tarball. I spent time trying to figure out how to dedupe but think I ended up wasting too much time on silly things and decided to just duplicate for now.

Reply:
No worries, I'll probably take a look at it at "some point in the future" if that's okay with you :saluting_face: